### ISSUE TITLE
Feature validation

### ISSUE THREAD

**Issue by ms609** [palaeoware/trevosim#63]
Per https://github.com/openjournals/joss-reviews/issues/6722

I've been working through the reviewer checklist – almost all the boxes are now ticked!

I am stuck on the **Automated tests** point, "the functionality of the software can be verified".  I couldn't see an obvious way to validate the Match Peaks and No Selection features.  Are these covered by existing tests, or is there a manual way to evaluate this?

**Comment by RussellGarwood** [comment]
Oh, thanks for catching this - I suspect both are missing having traled through the git history on the development repo, though I thought I had done them at some point. In theory you should be able to check both of these using the complete working log that outputs everything TREvoSim does, but this is unweildy and it would be much better to have tests - I shall implement some in coming days!

**Comment by RussellGarwood** [comment]
With above push, no selection mode is rolled into test 6, as it fundamentally works by modify the individual selected for replication. I also took the opportunity to, hopefully, clarify the reporting of the test, but given its nature  I suspect it will remain hard to follow without looking at the code!

**Comment by RussellGarwood** [comment]
Apologies, it looks like I forgot to comment on this issue (I've been travelling between Berlin and Manchester, and setting up a new computer, so it's all been a bit chaotic!) - the above commit adds a test for matching peaks by doing doing an exhaustive search and ensuring that the fittest individuals when match peaks is on have the same fitness (and that they don't across multiple repeats when it is off!).
